src/averageAges.js

```diff
@@ -37,7 +48,18 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const womanCheck = person => person.sex === 'f';
+  const motherCheck = motherName => {
+    const mother = people.find(person => person.mother === motherName);
+
+    return mother;
+  };
```

Comment:
What will be in case the function won't find anything?
```suggestion
  const motherCheck = (motherName) => 
    people.find(person => person.mother === motherName) || null;
```